vizro-ai/src/vizro_ai/dashboard/_response_models/components.py

```diff
@@ -63,18 +63,20 @@ def create(self, model: BaseChatModel, all_df_metadata: AllDfMetadata) -> Compon
                 from vizro_ai import VizroAI
 
                 vizro_ai = VizroAI(model=model)
-                result = vizro_ai._run_plot_tasks(
+                result = vizro_ai.plot(
                     df=all_df_metadata.get_df(self.df_name),
                     user_input=self.component_description,
-                    chart_name=self.component_id,
-                    max_debug_retry=2,
+                    max_debug_retry=2,  # TODO must be flexible
```

Comment:
Why do we want to make it flexible? I feel this control would be too granular to be controlled by dashboard users